modin/engines/base/frame/partition_manager.py

```diff
@@ -167,7 +152,7 @@ def get_partitions(index):
         return new_partitions
 
     @classmethod
-    def broadcast_apply(cls, axis, apply_func, left, right):
+    def broadcast_apply(cls, axis, apply_func, left, right, other_name="r"):
```

Comment:
Please add docstring label for `other_names`.

Reply:
done